setup.py

```diff
@@ -134,11 +141,6 @@ def check_python_version():
         )
 
     def additional_commands():
-        # Pygments 2 isn't supported on Python 3 versions earlier than 3.3, so
```

Comment:
We already have a python version check (`check_python_version`) that excludes these versions of Python, so this check is redundant.